### PR TITLE
feat(dam): Add feature badge

### PR DESCRIPTION
**Pull request by matejminar**
image

### CODE REVIEW

File: tests/js/spec/components/modals/addDashboardWidgetModal.spec.jsx [hunk 1]
```diff
@@ -1411,9 +1409,7 @@ describe('Modals -> AddDashboardWidgetModal', function () {
       });
 
       await tick();
-      await act(async () =>
-        userEvent.click(screen.getByLabelText('Metrics (Release Health)'))
-      );
+      await act(async () => userEvent.click(screen.getByLabelText(/metrics/i)));
```
Comment (priscilawebdev) [inline tests/js/spec/components/modals/addDashboardWidgetModal.spec.jsx:1412]: so we need act here?

File: tests/js/spec/components/modals/addDashboardWidgetModal.spec.jsx [hunk 2]
```diff
@@ -1445,9 +1441,7 @@ describe('Modals -> AddDashboardWidgetModal', function () {
...
```
Comment (priscilawebdev) [inline tests/js/spec/components/modals/addDashboardWidgetModal.spec.jsx:1444]: here too

File: tests/js/spec/components/modals/addDashboardWidgetModal.spec.jsx (hunk 1 above)
Comment (matejminar) [reply tests/js/spec/components/modals/addDashboardWidgetModal.spec.jsx:1412]: Looks like we do, because of changes to MetricsSearchBar.

**Review by priscilawebdev — APPROVED** [review]
Approving to unblock you

**Pull request merged**